dwc_otg: fix an undeclared variable

RussellWang9 opened the pull request:
In `drivers/usb/host/dwc_otg/dwc_otg_hcd_linux.c: endpoint_reset() line#1029`, there is an undeclared variable `epnum`. 
It's alright for now because 'DWC_DEBUGPL' dose nothing when `DEBUG` is not defined. But it goes wrong when you uncomment `ccflags-y	+= -DDEBUG`  in `drivers/usb/host/dwc_otg/Makefile: line#15`. Error output (undeclared variable) will occur when compiling the kernel.

pelwell commented:
The change itself looks OK to me, but can you resubmit your patch with a brief description mentioning DWC_DEBUGPL in the commit message, along with a "Signed-off-by:" line?

RussellWang9 committed:
dwc_otg: fix an undeclared variable
Replace an undeclared variable used by DWC_DEBUGPL with the real endpoint address. DWC_DEBUGPL does nothing with DEBUG undefined so it did not go wrong before.
Signed-off-by: Zixuan Wang <wangzixuan@sjtu.edu.cn>

wzx-ipads commented:
> The change itself looks OK to me, but can you resubmit your patch with a brief description mentioning DWC_DEBUGPL in the commit message, along with a "Signed-off-by:" line?

Sure.

pelwell commented:
I was just about to merge this PR - did you close it by accident?

pelwell commented:
Never mind - I see #4298.

wzx-ipads commented:
> I was just about to merge this PR - did you close it by accident?

Sorry for that! I made a mistake.
